banyan/resource_service_web.go

```diff
@@ -499,16 +538,30 @@ func expandCustomTLSCert(d *schema.ResourceData) service.CustomTLSCert {
 
 func expandWebHTTPSettings(d *schema.ResourceData) (httpSettings service.HTTPSettings) {
 	httpSettings = service.HTTPSettings{
-		Enabled:         true,
-		OIDCSettings:    expandWebOIDCSettings(d),
-		ExemptedPaths:   expandWebExemptedPaths(d),
-		Headers:         expandCustomHttpHeaders(d),
-		HTTPHealthCheck: expandWebHTTPHealthCheck(),
-		TokenLoc:        expandWebTokenLoc(d),
+		Enabled:           true,
+		CustomTrustCookie: expandCustomTrustCookie(d),
+		OIDCSettings:      expandWebOIDCSettings(d),
+		ExemptedPaths:     expandWebExemptedPaths(d),
+		Headers:           expandCustomHttpHeaders(d),
+		HTTPHealthCheck:   expandWebHTTPHealthCheck(),
+		TokenLoc:          expandWebTokenLoc(d),
 	}
 	return
 }
+func expandCustomTrustCookie(d *schema.ResourceData) *service.CustomTrustCookie {
+	v, ok := d.GetOk("custom_trust_cookie")
+	if !ok {
+		return nil
+	}
+	tc := v.(*schema.Set).List()
+	sameSitePolicy := tc[0].(map[string]interface{})["same_site_policy"].(string)
```

Comment:
do we need to check `len(tc) != 0`?

Reply:
Not required here because it would always have an element. Copied from old code. But I think better to add it for safety.